auraescript/src/lib.rs

```diff
@@ -67,8 +67,8 @@ mod builtin;
 mod cells;
 mod cri;
 mod discovery;
-mod health;
 mod observe;
+mod health;
```

Comment:
Nitpick: alphabetical

Sidenote: I think rustfmt would take care of this automatically, so you may not have it turned on.

Also, don't think of the nitpicks as blockers. I wouldn't block the merge just for those.

Reply:
I do have `rustfmt` enabled on save so I'm a bit puzzled as well as why it doesn't order these automatically.